apps/condo/domains/user/schema/ConfirmPhoneAction.js

```diff
@@ -179,6 +263,9 @@ const ConfirmPhoneActionService = new GQLCustomSchema('ConfirmPhoneActionService
                     throw new Error('[error]: Unable to find confirm phone action by token')
                 }
                 const { id, phone } = actions[0]
+                await checkDayLimitCounters(phone, context.req.ip)
+                await checkLock(phone, 'confirmPhoneActionResendSms')
```

Comment:
it should be the same lock key for start/resend/confirm

Reply:
Done